.github/workflows/test-package.yml

```
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        sdk: [ dev ]
+        sdk: [ 2.12.0, dev ]
```

```suggestion
        sdk: [ dev ]
```

You want to add 2.12 in the test step below!